src/m365/aad/commands/approleassignment/approleassignment-add.ts

```diff
@@ -235,7 +235,7 @@ class AadAppRoleAssignmentAddCommand extends GraphCommand {
       }
     };
 
-    return request.post(requestOptions);
+    return await request.post(requestOptions);
```

Comment:
Hi @MathijsVerbeeck, this in fact is not necessary. You can just return the promise without awaiting it here.